standard/standard_4.php

```diff
@@ -172,13 +171,12 @@ function unserialize (string $str, array $options = []) {}
 /**
  * Dumps information about a variable
  * @link https://php.net/manual/en/function.var-dump.php
- * @param mixed $expression <p>
+ * @param mixed ...$expressions <p>
  * The variable you want to export.
  * </p>
- * @param mixed $_ [optional]
  * @return void
  */
-function var_dump ($expression, $_ = null) {}
+function var_dump (...$expressions) {}
```

Comment:
it's not the same because it also means that I can use this function without any arguments but I can't. Could you also check other similar changes?

Reply:
You are right, checked and reverted. :+1: